app/controllers/circle/members_controller.rb

```diff
@@ -56,4 +57,27 @@ def active_users
       .order('last_name asc')
   end
 
+  def organizers
```

Comment:
Having a mutation for that will also allow reusing the code in other places (controllers).